dozer/cogs/voice.py

```diff
@@ -124,6 +183,45 @@ async def get_by(cls, **kwargs):
         return result_list
 
 
+class AutoPTT(db.DatabaseTable):
+    """DB object to keep track of voice to text channel access bindings."""
+    __tablename__ = 'autoptt'
+
+    __uniques__ = 'id'
+
+    @classmethod
+    async def initial_create(cls):
+        """Create the table in the database"""
+        async with db.Pool.acquire() as conn:
+            await conn.execute(f"""
+            CREATE TABLE {cls.__tablename__} (
+            id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
What's your thoughts on using the channel ID as the primary key? That should speed up the query in `on_PTT_check` and simplify the `autoptt` command.